nixie/unittests/test_render_diagram.py

```
@@ -88,6 +95,44 @@ async def fake_wait_for_proc(
     assert "mmdc" in msg
 
 
+@pytest.mark.asyncio
+async def test_render_diagram_merman_failure_reports_raw_stderr(
+    monkeypatch: pytest.MonkeyPatch,
+    tmp_path: Path,
+    fake_home_cwd: Path,
```

Ruff (0.15.15)

[warning] Unused function argument: `fake_home_cwd`

(ARG001)